### DISCUSSION TITLE
gitlab workflow

### DISCUSSION THREAD

**Discussion by savushkin-yauheni** [projectdiscovery/nuclei-templates#1727] [Q&A] [answered]
Hi team.

How right now can we execute gitlab workflow ?

```
nuclei -w workflows/gitlab-workflow.yaml -target https://google.com -tags fuzz --debug
```

```
[WRN] Could not parse file '/root/nuclei-templates/workflows/gitlab-workflow.yaml': tags filter not matched workflow
```

Thanks

**Comment by ehsandeep** [comment]
Unfortunately it can not in the current form due to how filter works, for the time being we will be removing fuzz tags from few related templates and fix in next nuclei release.

**Reply by ehsandeep** [reply, marked as answer]
will be usable after next release - https://github.com/projectdiscovery/nuclei-templates/commit/4160cb168a5360dcc6b10bf17d00fe82e7a21143
